packages/expo-camera/src/CameraView.tsx

```diff
@@ -175,24 +175,27 @@ export default class CameraView extends Component<CameraViewProps> {
   }
 
   /**
-   * Presents a modal view controller that uses the [`DataScannerViewController`](https://developer.apple.com/documentation/visionkit/scanning_data_with_the_camera) available on iOS 16+.
+   * On iOS, presents a modal view controller that uses the [`DataScannerViewController`](https://developer.apple.com/documentation/visionkit/scanning_data_with_the_camera) available on iOS 16+.
+   * On Android, will use the [`Google Code Scanner`](https://developers.google.com/ml-kit/vision/barcode-scanning/code-scanner).
    * @platform ios
+   * @platform android
```

Comment:
```suggestion
   * On Android, will use the [Google code scanner](https://developers.google.com/ml-kit/vision/barcode-scanning/code-scanner).
   * On iOS, presents a modal view controller that uses the [`DataScannerViewController`](https://developer.apple.com/documentation/visionkit/scanning_data_with_the_camera) available on iOS 16+.   
   * @platform android
   * @platform ios
```

- Let's reformat this to use preferred platform order when presenting multiple platform info as per our writing style guide.
- As per Android docs linked, let's use the same convention from their docs on how they present "Google code scanner" in our docs as well.